Xamarin.Forms.Platform.iOS/PlatformRenderer.cs

```diff
@@ -126,21 +51,14 @@ public override bool ShouldAutorotateToInterfaceOrientation(UIInterfaceOrientati
 
 		public override void ViewDidAppear(bool animated)
 		{
-			Platform.DidAppear();
 			base.ViewDidAppear(animated);
+			Platform.DidAppear();
 		}
 
 		public override void ViewDidLayoutSubviews()
 		{
 			base.ViewDidLayoutSubviews();
 			Platform.LayoutSubviews();
 		}
-
-		public override void ViewWillAppear(bool animated)
-		{
-			View.BackgroundColor = UIColor.White;
-			Platform.WillAppear();
-			base.ViewWillAppear(animated);
-		}
```

Comment:
This code isn't doing anything magical either. If you look at `Platform.WillAppear`, it only executes once due to `_appeared` (which never gets set to false once true). Setting `View.BackgroundColor` of the platform renderer can be done in `Platform` while it's being initialized. We're not changing `View.BackgroundColor` anywhere to justify setting it to white again.